deployments/manifests/configmap.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: chowkidar
+    group: com.stakater.platform
+    version: dev
+    provider: stakater
+  name: chowkidar
+data:
+  .chowkidar.yaml: |
+    handler:
```

Comment:
I think this would be a bit different but fine for now. We will update the config structure once we have a controller running

Reply:
yeah agreed; its just a sample to get things running